packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -4,14 +4,18 @@
 
 import 'dart:async';
 import 'dart:math';
+import 'dart:ui' show lerpDouble;
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/gestures.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/widgets.dart';
+import 'package:flutter/animation.dart' show Curves;
 
 const double _kBackGestureWidth = 20.0;
 const double _kMinFlingVelocity = 1.0; // Screen widths per second.
+const int _kMaxPageForwardAnimationTime = 800; // Milliseconds.
```

Comment:
Document where these values came from. (just saying they're eyeballed is fine too)